src/lava/magma/core/process/connection.py

```diff
@@ -62,11 +62,11 @@ def __init__(
         **kwargs,
     ):
         kwargs["learning_rule"] = learning_rule
-        kwargs["graded_input"] = graded_input
+        #kwargs["graded_input"] = graded_input
         kwargs["shape"] = shape
 
         self.learning_rule = learning_rule
-        self.graded_input = graded_input
+        #TODO move that somewhere else if needed self.graded_input = graded_input
```

Comment:
Same, this is just a flag to accept a graded input to the Dense process. What would be the alternative?